driver-core/src/main/com/mongodb/client/model/changestream/ChangeStreamDocumentCodec.java

```diff
@@ -43,7 +45,7 @@
 
         ClassModelBuilder<ChangeStreamDocument> classModelBuilder = ClassModel.builder(ChangeStreamDocument.class);
         ((PropertyModelBuilder<TResult>) classModelBuilder.getProperty("fullDocument")).codec(codecRegistry.get(fullDocumentClass));
-        ((PropertyModelBuilder<ResumeToken>) classModelBuilder.getProperty("resumeToken")).codec(RESUME_TOKEN_CODEC);
+        ((PropertyModelBuilder<RawBsonDocument>) classModelBuilder.getProperty("resumeToken")).codec(RESUME_TOKEN_CODEC);
```

Comment:
Since the API exposes this as a BsonDocument, there's no reason I can think of to use RawBsonDocument here.  Just use BsonDocument.